[Doc] Update 22.08 documentation (#6216)
NVIDIA/spark-rapids: docs/download.md

that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.

## Release v22.08.0
Hardware Requirements:

The plugin is tested on the following architectures:

	GPU Models: NVIDIA V100, T4 and A2/A10/A30/A100 GPUs

Software Requirements:

	OS: Ubuntu 18.04, Ubuntu 20.04 or CentOS 7, Rocky Linux 8

	CUDA & NVIDIA Drivers*: 11.x & v450.80.02+

> 	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.3.0, Databricks 9.1 ML LTS or 10.4 ML LTS Runtime and GCP Dataproc 2.0

Reviewer: ```suggestion
	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.2.2, 3.3.0, Databricks 9.1 ML LTS or 10.4 ML LTS Runtime and GCP Dataproc 2.0
```

	Python 3.6+, Scala 2.12, Java 8

*Some hardware may have a minimum driver version greater than v450.80.02+.  Check the GPU spec sheet
for your hardware's minimum driver version.

*For Cloudera and EMR support, please refer to the
[Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.

### Release Notes
New functionality and performance improvements for this release include:
* Rocky Linux 8 support

> * Ability to build Spark RAPIDS jar using JDK 11 

Reviewer: ```suggestion
* Ability to build Spark RAPIDS jars for Java versions 9+
```

> * ZSTD parquet read support

Reviewer: ```suggestion
* Zstandard Parquet and ORC read support
```

* Binary read support from parquet

> * Apache iceberg 0.14 support  

Reviewer 2: ```suggestion
* Apache Iceberg 0.13 support  
```

* Array function support: array_intersect, array_union, array_except and arrays_overlap

> * Function from_json support

Author: Removed.

> * Function from_json support

Author: Fixed all above.
